examples/voting/package.json

```diff
@@ -1,11 +1,11 @@
 {
     "name": "voting",
     "license": "Apache-2.0",
-    "version": "1.1.0",
+    "version": "1.1.1",
     "packageManager": "yarn@3.2.0",
     "dependencies": {
         "@concordium/browser-wallet-api-helpers": "^2.0.0",
-        "@concordium/web-sdk": "^3.3.1",
+        "@concordium/web-sdk": "^6.0.0",
```

Comment:
I think you "need" to do this as a resolution. Otherwise you'll get 2 SDK versions in play at the same time; this one and [another one](https://github.com/Concordium/concordium-browser-wallet/blob/main/packages/browser-wallet-api-helpers/package.json#L22) from `browser-wallet-api-helpers`. It works most of the time but could cause problems down the line.